python/ray/tests/test_runtime_env_working_dir.py

```diff
@@ -194,6 +194,7 @@ def reinit():
     @ray.remote
     def test_import():
         import file_module
+
```

Comment:
Unrelated change for this and `python/ray/tests/test_runtime_env_working_dir_3.py`?